apps/docs/scripts/search/sources/github-discussion.ts

```diff
@@ -25,16 +25,22 @@ export type DiscussionsResponse = {
   }
 }
 
+const appId = process.env.SEARCH_GITHUB_APP_ID ?? process.env.DOCS_GITHUB_APP_ID
```

Comment:
There are a bunch of environment variables that are differently named between Vercel and GitHub Actions. We should consolidate these later but I'm leaving them alone for now.